.github/workflows/ci.yaml

```diff
@@ -15,7 +15,7 @@ jobs:
       - name: Setup Go
         uses: actions/setup-go@v5
         with:
-          go-version: 1.22
+          go-version: 1.23
```

Comment:
There are go versions hardcoded in each workflow file.  It probably makes sense to update them all in this PR at once.